shared/libraries/config_protocol/include/config_protocol/config_protocol.h

```diff
@@ -41,14 +41,17 @@ inline auto ParamsDict(std::initializer_list<std::pair<const StringPtr, ObjectPt
 
 enum PacketType: uint8_t { getProtocolInfo = 0x80, upgradeProtocol = 0x81, rpc = 0x82, serverNotification = 0x83, invalidRequest = 0x84 };
 
+#pragma pack(push, 1)
```

Comment:
force alignment on 1 byte is strictly not needed because the fields are laid in that way that they cannot be unaligned except if some system would align on non standard sizes (other than 1, 2, 4, 8)

Reply:
I added this just in case.